odoo/addons/base/tests/test_test_suite.py

```diff
@@ -123,12 +124,14 @@ def _log_error(self, message):
         self.test_result.addError(self, (AssertionError, AssertionError(message), None))
 
     def _clean_message(self, message):
-        root_path = __file__.replace('/odoo/addons/base/tests/test_test_suite.py', '')
+        root_path = __file__.replace('\\', '/').replace('/odoo/addons/base/tests/test_test_suite.py', '')
+        python_path = contextlib.__file__.replace('\\', '/').replace('/contextlib.py', '')
+        message = message.replace('\\', '/')
```

Comment:
I cannot test that on windows but I think that using Pathlib may do the job. 

```suggestion
        root_path = Path(__file__).parents[4]  # removes /odoo/addons/base/tests/test_test_suite.py
        python_path = Path(contextlib.__file__).parent  # /usr/lib/pythonx.x, C:\\python, ...
```

Can you test this solution ?

Reply:
it works if `message = message.replace('\\', '/')` is moved to the end instead of removing it, otherwise the tests will complain about the remaining slashes. I'll make the change.